src/test/java/net/openhft/chronicle/queue/ReadWriteTest.java

```diff
@@ -140,13 +128,18 @@ public void testProceedWhenMetadataFileInitialized() throws IOException {
             }
         }).start();
 
-        // This should succeed after 200ms
+        // the below can happen if the race mitigation code in TableDirectoryListingReadOnly.init is exercised
+        // as a LongValue gets created before it can be assigned to a reference and be available to be closed
+        ignoreException("Discarded without closing");
```

Comment:
Ah yeah this, we can fix this with my comment above I believe

Reply:
No, because an exception can be thrown after the LongValue is created and before it is returned